composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.0",
```

Comment:
p.s : these stats are meaningless IMO, as the percentages of versions usage on websites is not quite the same as the percentage of usage of php versions and psysh usage.... especially as A LOT of providers are NOT upgrading and letting sites rot on old php versions.